app/jobs/timeline_events/after_marking_as_complete_job.rb

```diff
@@ -6,9 +6,27 @@ def perform(submission)
       # Refuse to process submissions from reviewed targets.
       return if submission.evaluation_criteria.exists?
 
+      user = submission.founders.first.user
```

Comment:
This creates high coupling between the PF system and xAPI. I would prefer to have a lot loosely coupled parts here. Maybe we could have some notifications system build in PF core and xAPI module should only listen on events and asynchronously process them (build xAPI statement & send it to LRS).